### PR TITLE
test: rework to check multi entrypoints works

### PR DESCRIPTION
**Pull request by digitalsadhu**
## JIRA Issue
[COREWEB-120](https://jira.finn.no/browse/COREWEB-120)
[COREWEB-181](https://jira.finn.no/browse/COREWEB-181)

## Status
**READY**

## Description
Verifies fix for a bug where only 1 entrypoint for each feed would be
included

## Todos
- [x] Tests

### CODE REVIEW

File: test/__snapshots__/scenarios.test.js.snap [hunk 1]
```diff
@@ -239,7 +239,122 @@ body {
...
...\"==typeof require&&require;if(!c&&f)return f(o,!0);if(e)return e(o,!0);var s=new Error(\\"Cannot find module '\\"+o+\\"'\\");throw s.code=\\"MODULE_NOT_FOUND\\",s}var p=r[o]={exports:{}};n[o][0].call(p.exports,function(t){var r=n[o][1][t];return i(r||t)},p,p.exports,t,n,r,u)}return r[o].exports}for(var e=\\"function\\"==typeof require&&require,o=0;o<u.length;o++)i(u[o]);return i}({1:[function(t,n,r){\\"use strict\\";t(\\"./d\\")(5,6)},{\\"./d\\":2}],2:[function(t,n,r){\\"use strict\\";n.exports=function(t,n){return t+n}},{}],3:[function(t,n,r){\\"use strict\\";t(\\"./c\\");spy(\\"h\\")},{\\"./c\\":1}],4:[function(t,n,r){\\"use strict\\";t(\\"./d\\")(1,2)},{\\"./d\\":5}],5:[function(t,n,r){arguments[4][2][0].apply(r,arguments)},{dup:2}],6:[function(t,n,r){\\"use strict\\";t(\\"./a\\");spy(\\"f\\")},{\\"./a\\":4}],7:[function(t,n,r){arguments[4][2][0].apply(r,arguments)},{dup:2}],8:[function(t,n,r){\\"use strict\\";t(\\"./d\\")(10,6)},{\\"./d\\":7}],9:[function(t,n,r){\\"use strict\\";t(\\"./d\\")(3,4)},{\\"./d\\":10}],10:[function(t,n,r){arguments[4][2][0].apply(r,arguments)},{dup:2}],11:[function(t,n,r){\\"use strict\\";t(\\"./b\\");spy(\\"g\\")},{\\"./b\\":9}]},{},[8,6,11,3]);"`;
```
Comment (SimenB) [inline test/__snapshots__/scenarios.test.js.snap]: not the easiest diff, but I trust you!

File: test/__snapshots__/scenarios.test.js.snap [hunk 2]
```diff
@@ -239,7 +239,122 @@ body {
...
+      function(t, n, r) {
+        \\"use strict\\";
+        t(\\"./b\\");
+        spy(\\"g\\");
+      },
+      { \\"./b\\": 7 }
+    ]
+  },
+  {},
+  [5, 9, 10, 3]
```
Comment (digitalsadhu) [inline test/__snapshots__/scenarios.test.js.snap]: 5 is layout server, 9 and 10 are multi-entry points for 1 podlet, 3 is another podlet with single entrypoint.

If you look through you can see that 9 and 10 share the same version of 2 where as 5 and 3 get their own versions though in none of the cases is 2 actually duplicated. `arguments[4][2][0].apply(r, arguments);` is used to make groups that all point back at 2.

File: test/__snapshots__/scenarios.test.js.snap (hunk 1 above)
Comment (digitalsadhu) [reply test/__snapshots__/scenarios.test.js.snap]: see my comment below

File: test/__snapshots__/scenarios.test.js.snap (hunk 2 above)
Comment (SimenB) [reply test/__snapshots__/scenarios.test.js.snap]: awesome

**Pull request merged**